File: Spring M2/Exception Handling/HrApp/src/main/java/com/cg/hrapp/dto/EmployeeRequestSubmitDTO.java
package com.cg.hrapp.dto;

import org.springframework.stereotype.Component;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Component
@Data
@AllArgsConstructor
@NoArgsConstructor
public class EmployeeRequestSubmitDTO {

	private String empName;
	private int salary;
	private String designation;
	private String location;
	private String panNumber;
	
	
}

File: pipe & validation ng/app/dto/employee-request-submit-dto.ts
export class EmployeeRequestSubmitDTO {
  empId: number;
  empName: string;
  salary: number;
  designation: string;
  location: string;
  panNumber: string;

  constructor(
    empId: number,
    empName: string,
    salary: number,
    designation: string,
    location: string,
    panNumber: string
  ) {
    this.empId = empId;
    this.empName = empName;
    this.salary = salary;
    this.designation = designation;
    this.location = location;
    this.panNumber = panNumber;
  }
}
